pandas-stubs/core/series.pyi

```diff
@@ -72,6 +72,26 @@ class Series(Generic[_DType]):
     def all(self, axis: int = ..., bool_only: bool = ...) -> bool: ...
     def any(self, axis: int = ..., bool_only: bool = ...) -> bool: ...
     @overload
+    def append(
+        self, to_append: Series, ignore_index: bool = ..., verify_integrity: bool = ...
+    ) -> Series: ...
+    @overload
+    def append(
+        self, to_append: List[Series], ignore_index: bool = ..., verify_integrity: bool = ...
+    ) -> Series: ...
+    @overload
+    def append(
+        self, to_append: Tuple[Series], ignore_index: bool = ..., verify_integrity: bool = ...
+    ) -> Series: ...
```

Comment:
This code is technically correct, but you can make it shorter with `Union`:

```python
def append(
        self, to_append: Union[Series, List[Series], Tuple[Series, ...]], ignore_index: bool = ..., verify_integrity: bool = ...
    ) -> Series: ...
```

You only need to use `overload` when the return type differs for different arguments.

So, here you need overload:

```python
@overload
def f(x: int) -> int: ...
@overload
def f(x: str) -> str: ...
```

But here you don't:
```python
@overload
def g(x: int) -> float: ...
@overload
def g(x: str) -> float: ...
```
you can do this instead:
```python
def g(x: Union[int, str]) -> float: ...
```

---

Also, you need to use `Tuple[Series, ...]` instead of `Tuple[Series]`. `Tuple[Series]` means a tuple with only **one** element. `Tuple[Series, ...]` means a tuple with an unknown number of elements.

You can also just use `Sequence[Series]`. `Sequence[T]` is basically equivalent to `Union[List[T], Tuple[T, ...]]`.